templates/tour.html

```diff
@@ -59,7 +59,7 @@ <h3>Simple device discovery</h3>
                     <li>Periodically</li>
                 </ul>
                 <p>
-                    <a href="https://docs.maas.io/release-notes"
+                    <a href="https://docs.maas.io/en/release-notes"
```

Comment:
Release notes seems like an odd place to "Learn more about device discovery and subnet mapping". But since this is definitely an improvement over what it was, I'm still :+1:ing the PR.

Reply:
Good point, update the link to a more relevant link.

Reply:
Cool. Thanks. Code +1 added again.